build_all_images.sh

```diff
@@ -242,88 +242,35 @@ function build_jar_runner_image {
-	then
-		echo ""
-		echo "Docker image JDK 8 is up to date."
-
-		return
-	fi
-
-	echo ""
-	echo "Building Docker image JDK 11/JDK 8."
-	echo ""
-
-	LIFERAY_DOCKER_IMAGE_PLATFORMS="${LIFERAY_DOCKER_IMAGE_PLATFORMS}" LIFERAY_DOCKER_REPOSITORY="${LIFERAY_DOCKER_REPOSITORY}" LIFERAY_DOCKER_ZULU_8_AMD64_VERSION=${latest_available_zulu8_amd64_version} LIFERAY_DOCKER_ZULU_8_ARM64_VERSION=${latest_available_zulu8_arm64_version} time ./build_jdk11_jdk8_image.sh "${BUILD_ALL_IMAGES_PUSH}" | tee -a "${LIFERAY_DOCKER_LOGS_DIR}"/jdk11_jdk8.log
-
-	if [ "${PIPESTATUS[0]}" -gt 0 ]
-	then
-		echo "FAILED: JDK 11/JDK 8" >> "${LIFERAY_DOCKER_LOGS_DIR}/results"
-
-		exit 1
-	else
-		echo "SUCCESS: JDK 11/JDK 8" >> "${LIFERAY_DOCKER_LOGS_DIR}/results"
-	fi
-}
-
-
-function build_jdk21_image {
-	local latest_available_zulu21_amd64_version=$(get_latest_available_zulu_version "21" "amd64")
-	local latest_available_zulu21_arm64_version=$(get_latest_available_zulu_version "21" "arm64")
+	local latest_available_zulu_amd64_version=$(get_latest_available_zulu_version "${jdk_version}" "amd64")
+	local latest_available_zulu_arm64_version=$(get_latest_available_zulu_version "${jdk_version}" "arm64")
 
-	if [[ $(get_latest_docker_hub_zulu_version "jdk21" "21" "amd64") == "${latest_available_zulu21_amd64_version}" ]] && [[ $(get_latest_docker_hub_zulu_version "jdk21" "21" "arm64") == "${latest_available_zulu21_arm64_version}" ]] && [[ "${LIFERAY_DOCKER_DEVELOPER_MODE}" != "true" ]]
+	if [[ $(get_latest_docker_hub_zulu_version "${jdk_image_name}" "${jdk_version}" "amd64") == "${latest_available_zulu_amd64_version}" ]] && [[ $(get_latest_docker_hub_zulu_version "${jdk_image_name}" "${jdk_version}" "arm64") == "${latest_available_zulu_arm64_version}" ]] && [[ "${LIFERAY_DOCKER_DEVELOPER_MODE}" != "true" ]]
 	then
 		echo ""
-		echo "Docker image JDK 21 is up to date."
+		echo "Docker image ${jdk_friendly_name} is up to date."
 
 		return
 	fi
 
 	echo ""
-	echo "Building Docker image JDK 21."
+	echo "Building Docker image ${jdk_friendly_name}."
 	echo ""
 
-	LIFERAY_DOCKER_IMAGE_PLATFORMS="${LIFERAY_DOCKER_IMAGE_PLATFORMS}" LIFERAY_DOCKER_REPOSITORY="${LIFERAY_DOCKER_REPOSITORY}" LIFERAY_DOCKER_ZULU_21_AMD64_VERSION=${latest_available_zulu21_amd64_version} LIFERAY_DOCKER_ZULU_21_ARM64_VERSION=${latest_available_zulu21_arm64_version} time ./build_jdk21_image.sh "${BUILD_ALL_IMAGES_PUSH}" | tee -a "${LIFERAY_DOCKER_LOGS_DIR}"/jdk21.log
+	LIFERAY_DOCKER_IMAGE_PLATFORMS="${LIFERAY_DOCKER_IMAGE_PLATFORMS}" LIFERAY_DOCKER_REPOSITORY="${LIFERAY_DOCKER_REPOSITORY}" LIFERAY_DOCKER_ZULU_AMD64_VERSION=${latest_available_zulu_amd64_version} LIFERAY_DOCKER_ZULU_ARM64_VERSION=${latest_available_zulu_arm64_version} time ./build_"${jdk_image_name//-/_}"_image.sh "${BUILD_ALL_IMAGES_PUSH}" | tee -a "${LIFERAY_DOCKER_LOGS_DIR}"/"${jdk_image_name}".log
```

Comment:
@Wykenz, please see https://github.com/natocesarrego/liferay-docker/pull/82#discussion_r1654851898 and https://github.com/natocesarrego/liferay-docker/commit/dcce0055420e8314aff997cbee278abe7aebbabe#r143653585.

Don't you need to make them compatible?